modules/nw-egress-ips-object-dual-stack.adoc

```diff
@@ -0,0 +1,148 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/assigning-egress-ips-ovn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-egress-ips-object-dual-stack_{context}"]
+= Configuring dual-stack networking for an EgressIP object
+
+For a cluster configured for dual-stack networking, you can apply dual-stack networking to a single `EgressIP` object. The `EgressIP` object that then extend dual-stack networking capabilities to a pod.
+
+////
+[IMPORTANT]
+====
+KNOWN ISSUE FROM https://issues.redhat.com/browse/OCPBUGS-61833
+====
+////
+
+The example in the procedure creates two `NodeNetworkConfigurationPolicy` (NNCP) custom resources (CRs) as a requirement for an egress IP configuration on a secondary node network interface controller (NIC) interface. One NNCP CR exists for each node that has assigned egress IP addresses where such nodes have a `k8s.ovn.org/egress-assignable` label.
```

Comment:
An important aspect worth mentioning is that both IPv4 and IPv6 addresses on the same EgressIP CR cannot be allocated to the same egress node, so there is a minimum requirement of two egress nodes.

Reply:
No. That's already documented elsewhere. If you want, add a reference to it.

Reply:
Thanks. I added a link to the prereq. I also moved the "Assigning an egress IP address to a namespace" section into the Configuring an egress IP address  document.